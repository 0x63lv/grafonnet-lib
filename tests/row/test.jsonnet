local grafana = import "grafonnet/grafana.libsonnet";
local row = grafana.row;

{
    basic: row.new(),
    advanced: row.new(
        title="Test Row",
        height="125px",
        collapse=true,
        repeat="env",
        showTitle=true,
        titleSize="h1"
    ),
}
