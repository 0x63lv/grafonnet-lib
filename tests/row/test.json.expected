{
   "advanced": {
      "collapse": true,
      "height": "125px",
      "panels": [ ],
      "repeat": null,
      "repeatIteration": null,
      "repeatRowId": null,
      "showTitle": true,
      "title": "Test Row",
      "titleSize": "h1"
   },
   "basic": {
      "collapse": false,
      "height": "250px",
      "panels": [ ],
      "repeat": null,
      "repeatIteration": null,
      "repeatRowId": null,
      "showTitle": false,
      "title": "Dashboard Row",
      "titleSize": "h6"
   },
   "panels": {
      "collapse": false,
      "height": "250px",
      "panels": [
         "foo",
         "bar"
      ],
      "repeat": null,
      "repeatIteration": null,
      "repeatRowId": null,
      "showTitle": false,
      "title": "Dashboard Row",
      "titleSize": "h6"
   }
}
